[DM-2784] [UI] Search results from OPC UA Address space can overflow the container (#1916)
SoftwareAG/c8y-docs: content/change-logs/device-management/cumulocity-10-20-0-0-fixed-opc-ua-address-space-device-tab-search-results-layout.md

---
date: ""

> title: Fixed search results overflow on small screens

Reviewer: ```suggestion
title: Search results on OPC UA Address space tab no longer overflow on small screens
```

product_area: Device management & connectivity
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:

>   - value: component--KIsStyzM

Reviewer: ```suggestion
  - value: component-Tf05_KQ-B 
```

>     label: Device Management app

Reviewer: ```suggestion
    label: OPC UA
```

build_artifact:

>   - value: tc-QHwMfWtBk7
>     label: cumulocity

Reviewer 2: ```suggestion
  - value: tc-pjJiURv9Y
    label: ui-c8y
```

ticket: DM-2784

> version: 1020.0.0

Reviewer 2: Is it agreed to not use exact version?

Author: Fixed

---

> Fixed OPC UA Address space device tab search results layout.

Reviewer: OPC UA Address space device tab > what does that exactly mean? Can you please provide a screenshot?

Author: Sure, there a video here: https://github.softwareag.com/IOTA/cumulocity-ui/pull/5882#issue-275182

> Fixed OPC UA Address space device tab search results layout.

Reviewer: ```suggestion
The layout of the search results in the **Address space** tab of an OPCUA device has been fixed.
```

> Fixed OPC UA Address space device tab search results layout.

Reviewer: ```suggestion
Proper search result layout on **Address space** tab of OPC UA devices
```

> Previously, when performing a search on small screen sizes, the search results would overflow the available space and extend beyond the visible area, making it difficult for users to view and interact with the results. This change addresses the overflow issue by ensuring the search results are properly contained within the available space on small screens. Users will now be able to easily view and navigate through the search results without any content being cut off, regardless of the screen size they are using.

Reviewer: ```suggestion
Previously, when performing a search in the **Address space** tab of OPC UA devices on small screen sizes, the search results would overflow the available space and extend beyond the visible area, making it difficult for users to view and interact with the results. This change addresses the overflow issue by ensuring the search results are properly contained within the available space on small screens. Users will now be able to easily view and navigate through the search results without any content being cut off, regardless of the screen size they are using.
```
